pkg/wharfyml/visit/mapvisitor.go

```diff
@@ -126,12 +131,15 @@ func (p MapVisitor) VisitStringSlice(key string, target *[]string) errutil.Slice
 	return errSlice
 }
 
+// VisitStringStringMap reads a node by string key and writes the string items
+// to the pointer. A slice of error contains any type errors. If the node is not
+// present, then nil is returned and the pointer is untouched.
```

Comment:
I feel this is clearer, I first thought of items as being just the value.
```suggestion
// VisitStringStringMap reads a node by string key and writes the string
// key-value pairs to the pointer. A slice of error contains any type errors. If
// the node is not present, then nil is returned and the pointer is untouched.
```